### PR TITLE
test: test easy install script periodically

### PR DESCRIPTION
**Pull request by rephus**
Closes: 

### Description:

https://github.com/lightdash/lightdash/actions/runs/4372642251/jobs/7649813179

image

### CODE REVIEW

File: .github/workflows/test-easy-install.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Debug env
+      run: env
+    - name: Run easy install
+      env: 
+        NODE_ENV: 'development'
```
[the remark this answers is not in the document]
Comment (ZeRego) [reply .github/workflows/test-easy-install.yml:22]: No, the install script is not relying on Node. We need to change the `track` function in the install script to be disabled via env var.

[the remark this answers is not in the document]
Comment (rephus) [reply .github/workflows/test-easy-install.yml:22]: right, ok, that will disable the tracking for the install script, but what about the lightdash.track that happens on the app ?

[the remark this answers is not in the document]
Comment (ZeRego) [reply .github/workflows/test-easy-install.yml:22]: We need to set all the env vars to empty. `RUDDERSTACK_WRITE_KEY` `RUDDERSTACK_DATA_PLANE_URL` `COHERE_TOKEN` `FULLSTORY_ORG_ID` `SENTRY_DSN`

[the remark this answers is not in the document]
Comment (rephus) [reply .github/workflows/test-easy-install.yml]: Setting '' RUDDERSTACK_WRITE_KEY, defaults to PROD key, so instead I'm using DEV key 

image

Before: 
Screenshot from 2023-03-10 10-33-08

now: 

image


Other keys are fine, they default to ''

**Review by ZeRego — APPROVED** [review]

**Pull request merged**